src/include/version.h

```diff
@@ -4,13 +4,16 @@
 #define LSMDATE "09Oct17"
 
 #define PROG_COPYRIGHT "Copyright (C) %d-2012 Free Software Foundation Europe e.V.\n" \
+                       "Copyright (C) 2010-2017 Planets Communications B.V.\n" \
```

Comment:
Can you explain the legal implication of this? Of course, I know you have been working at the Bareos code, also in the years 2010-2012. However, the commit dates start at 2013 (with your Bareos email address).
I'm especially unsure about the years 2010-2012, in which the copyright have been assigned to the FSFE. Does it align with copyright law to later claim the copyright?

Reply:
Its quite simple FSFE only got the patches which got upstreamed to Bacula as that is what
the FLA says. So all patches for BAREOS are my copyright and my copyright only for 2010, 2011 and 2012. Further more you keep also copyright yourself as the FLA reassigns the contributor to also license his code anyway he likes. But the most important is that all reassignment ONLY applies to things explicitly upstreamed to Bacula. If that would be true
anything I would have been doing even for my company would have its copyright transfered.
That is of course non-sense. B.T.W. the code already had this copyright claim only the generic
header not and the program output.